### PR TITLE
[docs][DOC-5] release notes for 2.12.12.0

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/releases/release-notes/v2.12/

### CODE REVIEW

File: docs/content/preview/releases/release-notes/v2.12.md [hunk 1]
```diff
@@ -17,6 +17,68 @@ Included here are the release notes for all releases in the v2.12 release series
...
 
+## v2.12.12.0 - February 10, 2023 {#v2.12.12.0}
+
+**Build:** `2.12.12.0-b12`
+
+**Third-party licenses:** [YugabyteDB](https://downloads.yugabyte.com/releases/2.12.12.0/yugabytedb-2.12.12.0-b12-third-party-licenses.html), [YugabyteDB Anywhere](https://downloads.yugabyte.com/releases/2.12.12.0/yugabytedb-anywhere-2.12.12.0-b12-third-party-licenses.html)
+
+### Downloads
+
+<ul class="nav yb-pills">
```
Comment (polarweasel) [inline docs/content/preview/releases/release-notes/v2.12.md:28]: It's worth doing a `curl -I <download-url>` for all three download packages and both license files before you publish, to make sure they actually exist.

File: docs/content/preview/releases/release-notes/v2.12.md [hunk 2]
```diff
@@ -17,6 +17,68 @@ Included here are the release notes for all releases in the v2.12 release series
...
+```sh
+docker pull yugabytedb/yugabyte:2.12.12.0-b12
+```
+
+### Improvements
+
+#### Yugabyte Platform
+
+[PLAT-6305] Upgrade Golang third party vulnerabilities (azcopy version to 10.16.2)
+[PLAT-6358] Allow defining prometheus remote write config in helm chart
```
Comment (polarweasel) [inline docs/content/preview/releases/release-notes/v2.12.md:57]: Bullets for all, everywhere
```suggestion
* [PLAT-6305] Upgrade Golang third party vulnerabilities (azcopy version to 10.16.2)
* [PLAT-6358] Allow defining prometheus remote write config in helm chart
```

File: docs/data/currentVersions.json [hunk 3]
```diff
@@ -40,9 +40,9 @@
     {
       "series": "v2.12",
       "display": "v2.12",
-      "version": "2.12.11.0",
-      "versionShort": "2.12.11",
-      "appVersion": "2.12.11.0-b34",
+      "version": "2.12.12.0",
+      "versionShort": "2.12.12",
+      "appVersion": "2.12.12.0-b10",
```
Comment (polarweasel) [inline docs/data/currentVersions.json:45]: Build 10, or 12?

**Review by polarweasel — APPROVED** [review]
One thing to fix, five URLs to try before you merge.

File: docs/content/preview/releases/release-notes/v2.12.md [hunk 4]
```diff
@@ -17,6 +17,68 @@ Included here are the release notes for all releases in the v2.12 release series
...
+
+#### Database
+
+[[13891](https://github.com/yugabyte/yugabyte-db/issues/13891)] Do not update index permissions when table is being deleted
+[[14417](https://github.com/yugabyte/yugabyte-db/issues/14417)] [DocDB] YB-Tserver Registration hazard - UUID can be blank
+[[15172](https://github.com/yugabyte/yugabyte-db/issues/15172)] [YSQL] Move the ybRunContext to be a child of TopPortalContext
+[[15698](https://github.com/yugabyte/yugabyte-db/issues/15698)] [DocDB] Handle null placement info due to faulty restores
+[[15820](https://github.com/yugabyte/yugabyte-db/issues/15820)] [YSQL] Fix ybgin index scan after DROP COLUMN
+[[15863](https://github.com/yugabyte/yugabyte-db/issues/15863)] Fixed race in Tablet::FlushWriteIndexBatch
+
```
Comment (polarweasel) [inline docs/content/preview/releases/release-notes/v2.12.md:81]: Should have known issues in here, too. (Looks like I forget them in some others!)
```suggestion

### Known issues

* N/A

```

File: docs/data/currentVersions.json (hunk 3 above)
Comment (aishwarya24) [reply docs/data/currentVersions.json:45]: sorry missed changing here .

**Pull request merged**